src/spectral/rulesets/.defaultsForSpectral.yaml

```
@@ -232,3 +232,15 @@ rules:
     - $.paths[*][*].requestBody.content[*].schema
     then:
       function: string-boundary
+
+  # ensure server variable has default value
+  server-variable-default-value:
+    description: 'Server variable should have default value'
+    message: "{{error}}"
```

`{{error}}` targets the error message from a custom function.

Here, we want to use the description as the error message.

```suggestion
    message: "{{description}}"
```